### PR TITLE
fix up some issues with akka naming

### PR DESCRIPTION
**Pull request by pjfanning**
* handles most remaining issues

### CODE REVIEW

File: docs/src/main/paradox/snapshots.md [hunk 1]
```diff
@@ -28,24 +27,22 @@ Maven
 
 sbt
 :   ```scala
-    resolvers += Resolver.sonatypeRepo("snapshots")
+    resolvers += Resolver.ApacheMavenSnapshotsRepo
```
Comment (mdedetrich) [inline docs/src/main/paradox/snapshots.md:30]: Can you also provide the pre sbt 1.9.0 variant? I am on my phone right now so I can't really provide examples easily but if you check my commits for using the Apache resolver constant you can find the langauge that I used

**Review by mdedetrich — APPROVED** [review]
Lgtm. Found one minor thing but it can also be fixed in future PR

**Review by He-Pin — APPROVED** [review]
lgtm

File: docs/src/main/paradox/cassandra.md [hunk 2]
```diff
@@ -308,7 +308,7 @@ pekko.projection.cassandra {
 }
 ```
 
-or share the same Cassandra session as [Alpakka Cassandra](https://doc.akka.io/docs/alpakka/2.0/cassandra.html):
+or share the same Cassandra session as [Pekko Connectors Cassandra](https://pekko.apache.org/docs/pekko-connectors/2.0/cassandra.html):
```
Comment (He-Pin) [inline docs/src/main/paradox/cassandra.md:311]: `2.0` is not accessible @pjfanning would you like to update this too?

Comment (pjfanning) [reply docs/src/main/paradox/cassandra.md:311]: thanks - I created https://github.com/apache/incubator-pekko-projection/pull/40

**Pull request merged**